### ISSUE TITLE
Avoid Etc.getlogin

### ISSUE THREAD

**Issue by someara** [test-kitchen/kitchen-digitalocean#42]
Hi.

I've ran into this a few times in the past, and have managed to hack around it, but...

The underlying C code in the Ruby class looks at the environment
```login = getenv("USER");```

This causes problems when running test-kitchen after scrubbing the environment, which is a common (and proper) thing to do when shelling out from $foolang

It's used here...

https://github.com/test-kitchen/kitchen-digitalocean/blob/89d73629a2cb5477c20d816738814e0b8dabd45a/lib/kitchen/driver/digitalocean.rb#L117

Thanks!

-s

**Comment by RoboticCheese** [comment]
@somera I no longer remember making a fix for this, but apparently I did in other drivers. Would changing it to `(Etc.getlogin || 'nologin')` like in kitchen-(openstack|rackspace) be sufficient?

Of course, it could go away entirely. It only works this way because I wrote the name generator while on a team of ~10 people who shared one cloud account and could never easily eyeball which servers were whose.
